katalog/prometheus-operator/rbac.yml

```diff
@@ -1,22 +1,43 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
```

Comment:
Are there any advantages in using labels on RBAC resources?

cc @angelbarrera92 @phisco

Reply:
Query advantages. If you want to query all resources created with this deployment you can just:

kubectl get all,secret,whatever -l app=myapp

Reply:
👍

Reply:
Ok, for now I would just stick to `app: <something>` if we all agree.